index.js

```diff
@@ -18,26 +18,42 @@ module.exports = fp(async function (app, { all, policy, ttl, cacheSize, skip, st
   onSkip = onSkip || noop
 
   let cache = null
+  let logTimer = null
+  let cacheReport = null
+
+  if (logInterval && policy.Query) {
+    initCacheReport()
+  }
 
   app.graphql.cache = {
     refresh () {
       buildCache()
-      setupSchema(app.graphql.schema, policy, all, cache, skip, storage, onHit, onMiss, onSkip)
+      setupSchema(app.graphql.schema, policy, all, cache, skip, storage, onHit, onMiss, onSkip, cacheReport)
     },
 
     clear () {
       cache.clear()
+      clearCacheReport()
     }
   }
 
   app.addHook('onReady', async () => {
     app.graphql.cache.refresh()
+    if (logInterval && policy.Query) {
+      logTimer = setInterval(logReport, logInterval * 1000).unref()
```

Comment:
I think this is seconds not minutes

Reply:
My bad. I guess I will change the documentation and change the interval format from `minutes` to `seconds`.